src/main/java/com/example/onlinebookstore/dto/shoppingcart/CartItemDto.java

```diff
@@ -0,0 +1,9 @@
+package com.example.onlinebookstore.dto.shoppingcart;
+
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Positive;
+
+public record CartItemDto(
+        @Positive @NotNull Long bookId,
+        @Positive @NotNull Integer quantity) {
+}
```

Comment:
This class should be request dto

```suggestion
public record CreateCartItemRequestDto(
        @Positive @NotNull Long bookId,
        @Positive @NotNull Integer quantity) {
}
```